forest-node/README.md

```diff
@@ -0,0 +1,43 @@
+
+## Running Forest In DigitalOcean
+
+## Forest IAC Architecture
+
+![Forest Cloud Infrastructure ](https://user-images.githubusercontent.com/47984109/219903795-77c306b8-a70b-4f32-8d7d-3c1a39e52186.jpg)
+
+## Requirements  
+- RAM: 16GB
+- VCPU: 2
+- Startup Disk Size: 200 GB
+- Expected Total Disk Size: > 500 GB
+- SSH Key should be created locally using `ssh keygen` and then added into digitalocean console where the fingerprint can be generated and added as a variable while creating the droplet.
+- Install terraform and ansible.  
```

Comment:
Could link to https://developer.hashicorp.com/terraform/downloads and https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html

Reply:
Got it.